pages/homePage/homePage.css

```diff
@@ -0,0 +1,197 @@
+@import "/styles.css";
+
+.page-wrap {
+
+    grid-template-areas:
+        'p_nav p_nav p_nav p_nav p_nav p_nav'
+        'p_content p_content p_content p_content p_content p_content'
+        'p_content p_content p_content p_content p_content p_content'
+    ;
+}
+
+/* Header */
+.page-header {
+    grid-area: p_header;
+}
+
+.page-sidebar {
+    display: none;
+}
+
+.page-footer {
+    display: none;
+}
+
+.main-content {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+.main-content__header {
+    height: 100%;
+}
+
+.promo-catagory-holder {
+    display: flex;
+    width: 100%;
+    height: 50%;
+    gap: 2rem;
+    padding: 0.8rem
+}
+
+.promo-catagory {
+    height: 100%;
+    width: auto;
+    aspect-ratio: 1/1;
+    margin: auto;
+    overflow: hidden;
+}
+
+.promo-catagory>img {
+    width: 100%;
+    height: auto;
+    transition: all .2s ease-in-out;
+}
+
+.promo-catagory>img:hover {
+    transform: scale(1.1);
+    opacity: 50%;
+}
+
+
+/* Carousal */
+
+/* Slideshow container */
+.slideshow-container {
+    height: 100%;
+    position: relative;
+    margin: auto;
+}
+
+/* Hide the slide by default */
+.slideshow-container__slide {
+    height: 100%;
+    overflow: hidden;
+}
+
+.slideshow-container__slide>img {
+    width: 100%;
+    height: auto;
+    object-fit: none;
+}
+
+/* Common properties of Next & Prev buttons */
+.slideshow-container__action-prev,
+.slideshow-container__action-next {
+    cursor: pointer;
+    position: absolute;
+    top: 50%;
+    width: auto;
+    margin-top: -2.2rem;
+    padding: 1.6rem;
+    color: var(--dui-light-theme-contrast-txt);
+    transition: 0.6s ease;
+    user-select: none;
+}
+/* Position the "next button" to the right */
+.slideshow-container__action-next {
+    right: 0rem;
+    border-radius: 3px 0px 0px 3px;
+}
+/* Position the "prev button" to the left */
+.slideshow-container__action-prev {
+    left: 0rem;
+    border-radius: 0px 3px 3px 0px;
+}
+
+/* On hover, add a black background color with a little bit see-through */
+.slideshow-container__action-prev:hover,
+.slideshow-container__action-next:hover {
+    background-color: rgba(0, 0, 0, 0.8);
+    color: var(--dui-light-theme-txt);
+}
+
+/* Caption text */
+.slideshow-container__slide-caption {
+    color: var(--dui-primary-color-p2-lgt-35pct);
+    font-size: 1.5rem;
+    padding: 0.8rem 1.2rem;
+    position: absolute;
+    bottom: 0.8rem;
```

Comment:
this value can be in px too

Reply:
My functionality as of now requires it to be rem